include/Global.h

```diff
@@ -7,6 +7,9 @@
 #include "AMR.h"
 #ifdef SUPPORT_LIBYT
 #include <libyt.h>
+#ifdef SUPPORT_LIBYT_INTERACTIVE
+#include <libyt_interactive_mode.h>
+#endif
```

Comment:
Move it to `include/GAMER.h` ([here](https://github.com/gamer-project/gamer/blob/main/include/GAMER.h#L57)). Also, check my general comment about including `libyt_interactive_mode.h` inside `lbiyt.h`.

Reply:
I will consider this later, currently we (Matt and I) are working on Enzo, and there are other minor things to do.
Do you want to put it as an individual issue in libyt?